src/SFA.DAS.ApprenticeCommitments.Api.AcceptanceTests/Features/RegisterRenewConfirm.cs

```diff
@@ -0,0 +1,70 @@
+﻿using AutoFixture;
+using FluentAssertions;
+using NUnit.Framework;
+using SFA.DAS.ApprenticeCommitments.Api.Controllers;
+using SFA.DAS.ApprenticeCommitments.Application.Commands.ChangeApprenticeshipCommand;
+using SFA.DAS.ApprenticeCommitments.Application.Commands.CreateRegistrationCommand;
+using SFA.DAS.ApprenticeCommitments.Application.Commands.VerifyRegistrationCommand;
+using SFA.DAS.ApprenticeCommitments.DTOs;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Mail;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.ApprenticeCommitments.Api.AcceptanceTests.Features
+{
+    public class RegisterRenewConfirm
+    {
+        private Fixture fixture;
+
+        [SetUp]
+        public void Setup()
+        {
+            fixture = new Fixture();
+        }
+
+        [Test]
+        public async Task Test()
+        {
+            var factory = Bindings.Api.CreateApiFactory();
+            var client = factory.CreateClient();
+            var db = Bindings.Database.CreateDbContext();
+
+            var create = fixture.Build<CreateRegistrationCommand>()
+                .With(p => p.Email, (MailAddress adr) => adr.ToString())
+                .Create();
+
+            var r1 = await client.PostValueAsync("apprenticeships", create);
+            r1.EnsureSuccessStatusCode();
+
+            var verify = fixture.Build<VerifyRegistrationCommand>()
+                .With(x => x.ApprenticeId, create.ApprenticeId)
+                .With(p => p.Email, create.Email)
+                .Create();
+
+            var r2 = await client.PostValueAsync("registrations", verify);
+            r2.EnsureSuccessStatusCode();
+
+            var (r3, apprenticeships) = await client.GetValueAsync<List<ApprenticeshipDto>>($"apprentices/{create.ApprenticeId}/apprenticeships");
+            var apprenticeshipId = apprenticeships[0].Id;
+
+            var r4 = await client.PostValueAsync(
+                $"apprentices/{create.ApprenticeId}/apprenticeships/{apprenticeshipId}/EmployerConfirmation",
+                new ConfirmEmployerRequest { EmployerCorrect = true });
+            r4.EnsureSuccessStatusCode();
+
```

Comment:
As we discussed in a thread, my concern here is if the user presses the button after the ChangeApprenticeshipCommand is executed, it means he will update the new commitment statement. Even through he has never viewed this statement. 

This is totally different from updating the commitment statement he has viewed

Reply:
That would be a good scenario to test in this manner I think.